packages/veritone-react-common/src/components/TranscriptEngineOutput/index.js

```diff
@@ -84,6 +90,11 @@ export default class TranscriptEngineOutput extends Component {
     };
   }
 
+  handleUserEditChange = evt => {
+    this.props.onToggleUserEditedOutput &&
```

Comment:
i've noticed people doing this often, I originally had the assumption that this function was optional, but i'm not so sure. If it's not optional then declare the prop as required and remove this validation as it isn't necessary.

Reply:
it is optional prop.